tests/unit/features/partitions/executor/test_executor.py

```diff
@@ -85,6 +107,14 @@ def test_clean(self, new_dir, partitions):
             (prime_dir / "primed-file").exists() is False for prime_dir in prime_dirs
         )
 
+        # Test symlinks for alias to default partition
+        if info.stage_alias_symlink:
+            assert info.stage_alias_symlink.exists() is False
```

Comment:
Same idea, but we fail if something unexpected changes in the future.

```suggestion
        assert info.stage_alias_symlink is None or info.stage_alias_symlink.exists() is False
```